src/schema/types/payment.graphql

```diff
@@ -0,0 +1,36 @@
+# Types
+type checkoutResult {
+    id: ID!
+    productId: String!
+    paymentId: Int!
+    status: String!
+    statusDetail: String!
+    dateApproved: Date!
+    paymentMethodId: String!
+    paymentTypeId: String!
+    payerEmail: String!
+}
+
+input CheckoutInput {
+  productId: ID!
+  paymentMethod: MercadoPagoInput!
+  amount: Float!
```

Comment:
The amount should be calculated by the backend. This should not be sent by the frontend.

Reply:
Done